gusto/timeloop.py

```diff
@@ -55,14 +56,18 @@ class Timestepper(BaseTimestepper):
     :arg forcing: a :class:`.Forcing` object
     """
 
-    def __init__(self, state, advection_dict, linear_solver, forcing, diffusion_dict=None):
+    def __init__(self, state, advection_dict, linear_solver, forcing, diffusion_dict=None, physics_list=None):
 
         super(Timestepper, self).__init__(state, advection_dict)
```

Comment:
Given that you changed the superclass constructor, you should now pass `diffusion_dict` into the constructor here too.